src/test/scala/com/databricks/spark/csv/CsvSuite.scala

```diff
@@ -197,9 +204,44 @@ abstract class AbstractCsvSuite extends FunSuite with BeforeAndAfterAll {
       .select("Name")
       .collect().size
 
+    val r = new CsvParser()
+                  .withSchema(strictSchema)
+                  .withUseHeader(true)
+                  .withParserLib(parserLib)
+                  .withParseMode(ParseModes.DROP_MALFORMED_MODE)
+                  .csvFile(sqlContext, ageFile)
+                  .select("Name")
+                  .collect()
```

Comment:
Hm.. I took another look. It seems this `val r` is not related with this PR and not used?